test/testdata/deployednettemplates/recipes/generate-recipe/generate_network.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+import argparse
+import simplejson
```

Comment:
I'd rather stick to standard library and I don't see simplejson doing anything special over standard library json